style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
> which makes code misleading since you assume it will be using the defining scope's this value.

Is this true? I don't make that assumption. I've worked on projects where the rule was to simply always use the fat arrow. For some reason I always thought that was a bad idea but I have never actually ran into a time where it caused an issue.

Is there ever a case where this would actually cause a problem?